src/shared_memory_manager.cc

```diff
@@ -346,12 +346,26 @@ SharedMemoryManager::GetMemoryInfo(
         std::string("Unable to find shared memory region: '" + name + "'")
             .c_str());
   }
+
+  // populate 'shm_mapped_addr'
+  size_t max_offset = 0;
   if (it->second->kind_ == TRITONSERVER_MEMORY_CPU) {
     *shm_mapped_addr = (void*)((uint8_t*)it->second->mapped_addr_ +
                                it->second->offset_ + offset);
+    max_offset = it->second->offset_;
   } else {
     *shm_mapped_addr = (void*)((uint8_t*)it->second->mapped_addr_ + offset);
```

Comment:
Will `it->second->offset_` be zero for the `it->second->kind_ != TRITONSERVER_MEMORY_CPU` case? If so, can these cases be condensed into a single one to both do:
```cpp
    *shm_mapped_addr = (void*)((uint8_t*)it->second->mapped_addr_ +
                               it->second->offset_ + offset);
```

Reply:
The `it->second->offset_` is ignored when summing the `shm_mapped_addr` for `it->second->kind_ != TRITONSERVER_MEMORY_CPU` case. I wouldn't assume `it->second->offset_` to be zero for that case, because if it is always zero, then the both cases should have been condensed from the beginning.

@GuanLuo do you recall why the CPU shared memory address is calculated as
```
original address + offset when memory is registered + additional offset when inferencing
```
while the GPU shared memory address is 
```
original address                 +                    additional offset when inferencing
```
?

Reply:
I don't know, perhaps the address pointer has taken offset into account for CPU?

Reply:
@rmccorm4 I think lets not change it in this PR? In case something unexpected breaks?

Reply:
Sure no problem, was just curious. No need to follow up on it unless you find it useful later.